backend/spec/models/orgUserModel.spec.js

```diff
@@ -1,59 +1,75 @@
-  // beforeEach(async () => {
-  //   await mongoose.connection.collections.users.drop(() => {});
-  // });
+  beforeAll(async () => await db.connect());
+  beforeEach(async () => await db.clear());
+  afterAll(async () => await db.close());
 
-  it("has an email address", () => {
+  it("has the organisation name", () => {
     const orgUser = new OrgUser({
       organisationName: "Charity",
-        email: "someone@example.com",
-        charityNumber:"123456",
-        password: "password",
+      email: "someone@example.com",
+      charityNumber: "123456",
+      password: "password",
     });
 
-    expect(orgUser.email).toEqual("someone@example.com");
+    expect(orgUser.organisationName).toEqual("Charity");
   });
 
-  it("has a password", () => {
+  it("has an email address", () => {
     const orgUser = new OrgUser({
       organisationName: "Charity",
-        email: "someone@example.com",
-        charityNumber:"123456",
-        password: "password",
+      email: "someone@example.com",
+      charityNumber: "123456",
+      password: "password",
     });
-    expect(orgUser.password).toEqual("password");
-  });
 
-  it("has a organisation name", () => {
-    const orgUser = new OrgUser({
-        organisationName: "Charity",
-        email: "someone@example.com",
-        charityNumber:"123456",
-        password: "password",
-    });
-    expect(orgUser.organisationName).toEqual("Charity");
+    expect(orgUser.email).toEqual("someone@example.com");
   });
- 
+
   it("accepts a charity number", () => {
     const orgUser = new OrgUser({
-        organisationName: "Charity",
-        email: "someone@example.com",
-        charityNumber: "123456",
-        password: "password",
+      organisationName: "Charity",
+      email: "someone@example.com",
+      charityNumber: "123456",
+      password: "password",
     });
     expect(orgUser.charityNumber).toEqual(123456);
   });
 
-  it("accepts a blank charity number", () => {
+  it("has a password", () => {
     const orgUser = new OrgUser({
+      organisationName: "Charity",
+      email: "someone@example.com",
+      charityNumber: "123456",
+      password: "password",
+    });
+    expect(orgUser.password).toEqual("password");
+  });
+
+  describe("Returns error messages when input fields are not filled out correctly", () => {
+    it("accepts a blank charity number", () => {
+      const orgUser = new OrgUser({
         organisationName: "Organisation",
         email: "someone@example.com",
         charityNumber: "",
         password: "password",
+      });
+      expect(orgUser.charityNumber).toEqual(null);
+    });
+
+    it("returns error message when any of the fields that are 'required' is left blank", () => {
+      const orgUser = new OrgUser({
+        organisationName: "",
+        email: "",
+        charityNumber: "123456",
+        password: "",
+      });
+      expect(orgUser.error).not.toEqual("");
     });
-    expect(orgUser.charityNumber).toEqual(null);
```

Comment:
Are we testing this elsewhere (that the account is created even if charity number is left blank)?

Reply:
Yes we tested that an orgUser can created an account with charity number and another test where orgUser can create account without charity number